charts/unified-chart/templates/cronjobs.yaml

```diff
@@ -66,7 +59,7 @@ spec:
           volumes:
             - name: vol-secret
               secret:
-                secretName: {{ include "external-secret-unique-name" $ }}
+                secretName: {{ include "name-app-version" $ }}
```

Comment:
$ -> why not use the $top or something?
do you think we need to do something more explicit 
like 
```
{{ include "secret-name"  $ }}
```
that will use "name-app-version", if needed we'll change only one place that won't affect all others